DALI.tex

```diff
@@ -696,6 +696,13 @@ \subsection{Intervals}
 \end{verbatim}
 \noindent where \verb|short| or \verb|long| are also permitted.
 
+Multiple intervals may be serialised in a single value and described as (possibly) multiple
+with the following VOTable type metadata:
+
+\begin{verbatim}
+datatype="double" arraysize="*" xtype="interval"
+\end{verbatim}
```

Comment:
Should explicitly accommodate float and integer type possibilities here, since it's done elsewhere.